index.html

```diff
@@ -518,6 +518,174 @@ <h2>Thank you!!</h2>
             </section>
 
 
+           <!-- Lesson 2 -->
+            <section>
+                <h1>Introduction to Python</h1>
+                <h3>Lesson 2</h3>
+                <p>
+                    <small>Closely following <a href="https://automatetheboringstuff.com/">Automate the boring stuff with Python</a></small>
+                </p>
+            </section>
+
+             <!-- Booleans -->
+            <section title="Booleans">
+                <section>
+                    <h2>Booleans</h2>
+                    <p> Boolean values </p>
+                    <br>
+                    <small>The Boolean data type has only two values:</small>
+                    <br>
+                    <small>True and False</small>
+                    <small>Boolean is capitalized because the data type is named after mathematician George Boole</small>
+                </section>
+
+                <section>
+                    <h2>Booleans</h2>
```

Comment:
As discussed, can you remove the title as it's clear it's still about Booleans due to vertical slides